### DISCUSSION TITLE
Negative electrode potential

### DISCUSSION THREAD

**Discussion by aminacoric** [pybamm-team/PyBaMM#2948] [Q&A]
I noticed from the dynamic plot that the negative electrode potential is always 0 during a charge/discharge cycle. (The same goes for when I use the SPMe and DFN models).
Why is the negative electrode potential zero? 
Can you simulate the individual electrode potentials differently?

**Comment by brosaplanella** [comment]
The origin of potential is defined to be 0 V at the negative electrode - current collector interface, that's why the potential of the negative electrode is always close to 0 V. You can instead measure the potential difference between electrode and electrolyte, which would probably give the result you are expecting.

**Comment by Tobi0211H** [comment]
Hi,
as described above, the potential of the negative electrode refers to the negative current collector. This is understandable. My further questions would be:

1.  What is the reference for the electrolyte potential?
2. How can I get the anode potential related to Li/Li+ without the explicit use of plating in my model? 
I have read the example 
#https://docs.pybamm.org/en/latest/source/examples/notebooks/models/lithium-plating.html. 
But according to my understanding (I may be wrong), the anode potential related to Li/Li+ should be available even without modeling the plating. If not, could you please explain me why this is so?
Thank you very much for your support.
